Remotion/Data/DomainObjects.UnitTests/IntegrationTests/TableValuedParametersIntegrationTest.cs

```diff
@@ -178,6 +178,22 @@ public void Contains_UsingTableValuedParameter_DateTime ()
   {
     var deliveryDates = new[] { new DateTime(2005,2,1), new DateTime(2013,3,7) };
 
+    var query = QueryFactory.CreateQuery(
+        new QueryDefinition("Test", TestDomainStorageProviderDefinition, "SELECT * FROM [Order] o WHERE o.[DeliveryDate] IN (SELECT [Value] FROM @1)", QueryType.CollectionReadOnly),
```

Comment:
We should update the selection between DateTime and DateOnly TVP queries to target either a DateTime (for test on using DateTime) or a DateOnly column (for test on DateOnly). RIght now, both target Order.DeliveryDate which is either a DateTime or a DateOnly column.

Reply:
It's a DAteTime column. I'm not sure where and how I should add a DateOnly column. I tried, but it didn't create the column. Input?

Reply:
I would suggest we change the query to a table that has a Date-only column for querying date-only. (ClassWithAllDataTypes or the type where you updated the backing type to DateOnly - I think there is one in the PR?). The selected data is not relevant, only that the query qorks and we verify the result rows. There's no meaning beyond a successful where-condition filter.